Documentation/proposals/20250122-blackbox-crd.md

```diff
@@ -0,0 +1,52 @@
+## Your Proposal Title
+
+* **Owners:**
+  * [Pger-Y](https://github.com/Pger-Y)
+
+* **Related Tickets:**
+  * [Discussion](https://github.com/prometheus-operator/prometheus-operator/discussions/7134#discussioncomment-11569949)
+
+* **Other docs:**
+  * n/a
+
+This document proposes the creation of a Blackbox Operator and associated CRDs, similar to Prometheus-Operator, to manage blackbox-exporter instances and introduce a Module CRD for handling dynamic configuration of blackbox modules in a Kubernetes-native way.
```

Comment:
I don't see a reason for this to be a new operator. prometheus-operator already manages AlertManager and ThanosRuler resources for example.

Reply:
I think he means operator in sense of adding this capability to the actual prometheus operator, this might be confuse especially looking for our structure, where you have operator.go inside many different resources like Prometheus, Thanos and many others.